internal/pkg/template/template.go

```diff
@@ -415,3 +436,32 @@ func (t *Test) Inputs(provider testhelper.EnvProvider, replaceEnvsFn func(string
 
 	return inputs, nil
 }
+
+func (t *Test) SaveInputs(inputs map[string]templateInput.Value) error {
+	// Convert to Json
+	jsonContent, err := json.EncodeString(inputs, true)
+	if err != nil {
+		return err
+	}
+
+	// Write file
+	f := filesystem.NewRawFile(InputsFile, jsonContent)
+	return t.fs.WriteFile(f)
+}
+
+func (t *Test) SaveExpectedOutput(fs filesystem.Fs) error {
+	// Get expected output dir
+	if !t.fs.IsDir(ExpectedOutDirectory) {
+		err := t.fs.Mkdir(ExpectedOutDirectory)
+		if err != nil {
+			return err
+		}
+	}
+
+	originalFs, err := t.fs.SubDirFs(ExpectedOutDirectory)
```

Comment:
@webrouse tak jsem zase v koncích. 😅 Prosím tě, tady v `t.fs` by měl být adresář s konkrétním testem `tests/[testName]`, ale mně se tam dostává jeho nadadresář `tests` a potom se `expected-out` vytváří rovnou v něm. A už teda netuším kde dál hledat problém. 😕 

Funkcionalita vytvoření testu je pohromadě v tomhle commitu: https://github.com/keboola/keboola-as-code/pull/783/commits/89e1a62f64b9b63bbb4fa0ff64bc0e0fef40d56a tak to půjde asi nejlíp vidět tam. Tady ten `template.CreateTest` (https://github.com/keboola/keboola-as-code/blob/89e1a62f64b9b63bbb4fa0ff64bc0e0fef40d56a/pkg/lib/operation/template/local/test/create/operation.go#L37) dělá to samý co `template.Test` jen nevyfailuje když neexistuje adr. pro test a vyrobí ho. Potom tady chci uložit FS s projektem do `expected-out` toho testu (https://github.com/keboola/keboola-as-code/blob/89e1a62f64b9b63bbb4fa0ff64bc0e0fef40d56a/pkg/lib/operation/template/local/test/create/operation.go#L67) ale namísto do `tests/[testName]/expected-out` se uloží do `tests/expected-out`. 


![CleanShot 2022-09-02 at 17 42 12](https://user-images.githubusercontent.com/215660/188191663-f379427d-dc71-4d72-b8ca-0fec30887b59.jpg)

Reply:
Obzvlášť toto jsem kontroloval několikrát že se vrací `testDir` a ne `testsDir`. 😄
![CleanShot 2022-09-02 at 18 02 25](https://user-images.githubusercontent.com/215660/188193980-907f40ea-043e-4fe2-bd3b-e893912430ed.jpg)

Reply:
:eyes:

Reply:
Na prvy pohlad mi napada iba to, ze tak by sa to (mozno) chovalo, keby bolo `testName` prazdne.
Idem to vyskusat.

Reply:
💡aha, to by mohlo být. Zkusím, díky!

Reply:
jj, je to tak, pridam podmienku do `SubFs` aby to bola chyba:
![image](https://user-images.githubusercontent.com/19371734/188394690-6576d6ef-50b5-490a-b7cc-ddf682e294ba.png)

Reply:
PR: https://github.com/keboola/keboola-as-code/pull/787